documentation/book/assembly-access-configuration-kafka-mirror-maker.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-access-configuration-kafka-mirror-maker-{context}']
+= Configuring Kafka Mirror Maker authentication
+
+Configure Kafka Mirror Maker authentication using supported mechanisms:
+
+* TLS encryption
+* TLS client authentication
+* SASL SCRAM-SHA-512 and PLAIN authentication
+
+For more information on these mechanisms, see xref:con-kafka-mirror-maker-authentication-{context}[Authentication].
+
+include::proc-configuring-kafka-mirror-maker-tls.adoc[leveloffset=+1]
```

Comment:
of course I would remove even the reference to this one.

Reply:
Can I change to:  'For more information on the authentication mechanisms, see xref:con-kafka-mirror-maker-authentication-{context}[Authentication].'

Reply:
yes